src/bootstrap-table.js

```diff
@@ -3150,16 +3150,30 @@
       this.updatePagination()
     }
 
-    scrollTo (_value) {
-      if (typeof _value === 'undefined') {
+    scrollTo (params) {
+      if (typeof params === 'undefined') {
         return this.$tableBody.scrollTop()
       }
 
-      let value = _value
-      if (typeof _value === 'string' && _value === 'bottom') {
-        value = this.$tableBody[0].scrollHeight
+      let options = {unit: 'px', value: 0}
+      let scrollTo = 0;
```

Comment:
Other `;`, can use `yarn lint --fix`

Reply:
can remove this line

Reply:
Never used lint before, thank your for the tip.